src/runtime_src/core/pcie/tools/xbmgmt/xmc.cpp

```diff
@@ -500,15 +503,48 @@ int XMC_Flasher::writeReg(unsigned RegOffset, unsigned value) {
     return 0;
 }
 
+static std::string getStatus(int status, std::map<int, std::string> &map)
+{
+    auto entry = map.find(status);
+
+    return entry != map.end() ?
+        "0x" + std::to_string(entry->first) + "(" + entry->second + ")":
+	"0x" + std::to_string(entry->first);
+}
+
+/* XMC means CMC */
+void XMC_Flasher::initCMCStatusMap()
```

Comment:
Maybe make this an enum?

Reply:
We need to print the string like "NOT READY" or "READY". You mean we convert the 0,1,2,4 to enum?

Reply:
I see. This looks good.